rootfs/api/models/app.py

```diff
@@ -129,7 +132,65 @@ def delete(self, *args, **kwargs):
         self._clean_app_logs()
         return super(App, self).delete(*args, **kwargs)
 
-    def restart(self, **kwargs):
+    def restart(self, **kwargs):  # noqa
+        """
+        Restart found pods by deleting them (RC will recreate).
+        Wait until they are all drained away and RC has gotten to a good state
+        """
+        try:
+            # Fetch the initial set of pods to work from
+            pods = self.list_pods(**kwargs)
+            desired = len(pods)
+        except KubeException:
+            # Nothing was found
+            return []
+
+        try:
+            for pod in pods:
```

Comment:
I'm trying to avoid Threading if I can. So the bigger question is if we want to move this to a `watch` / `event` scrapping instead of doing it this way / threading